server/routes/slack/slack_events_helpers.py

```
@@ -59,6 +59,48 @@ def verify_slack_signature(request_body: bytes, timestamp: str, signature: str)
+
+
+def _lookup_slack_token(team_id: str, select_cols: str = "user_id") -> list:
+    """
+    Query user_tokens for Slack entries matching team_id, iterating orgs to
+    bypass FORCE ROW LEVEL SECURITY (which blocks even superuser without context).
+    """
+    org_ids = _get_all_org_ids()
+    for org_id in org_ids:
+        try:
+            with db_pool.get_admin_connection() as conn:
+                with conn.cursor() as cursor:
```

Ruff (0.15.15)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)